app-catalog/src/settings.tsx

```diff
@@ -0,0 +1,106 @@
+import { HoverInfoLabel, NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Box, Switch, SwitchProps, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+import { store } from './components/charts/List';
+
+export function AppCatalogSettings() {
+  const config = store.get();
+
+  const [currentConfig, setCurrentConfig] = useState(config);
+
+  function handleSave(value) {
+    const updatedConfig = { showOnlyVerified: value };
+    store.set(updatedConfig);
+    setCurrentConfig(store.get());
+    return;
+  }
+
+  function toggleShowOnlyVerified() {
+    const newShowOnlyVerified = currentConfig?.showOnlyVerified;
+    handleSave(!newShowOnlyVerified);
+  }
+
+  return (
+    <Box
+      style={{
+        marginTop: '3rem',
+      }}
+    >
+      <Typography variant="h5">App Catalog Settings</Typography>
+      <NameValueTable
+        rows={[
+          {
+            name: (
+              <HoverInfoLabel
+                label="Only verified"
+                hoverInfo="Show charts from verified publishers only"
+              />
+            ),
+            value: (
+              <EnableSwitch checked={!!currentConfig?.showOnlyVerified} onChange={toggleShowOnlyVerified} />
+            ),
+          },
+        ]}
+      />
+    </Box>
+  );
+}
+
+const EnableSwitch = (props: SwitchProps) => {
```

Comment:
EnableSwitch.tsx and EnableSwitch.stories.tsx in a separate commit.

Reply:
done!